Fixed issue #273 Add examples for dateFormatter

ManrajGrover opened the pull request:
@jzaefferer sir @rxaviers sir ,

I have added the examples for all of the available skeletons given in [this](https://github.com/ibm-js/ecma402/blob/master/cldr/en/ca-gregorian.json#L224) link. This fixes issue #273 .Please review and let me know of changes, if any.

jzaefferer commented:
This looks good to me. There are more issues with the document though, which I've reported in #422 - maybe something you could work on as well?

ManrajGrover commented:
@jzaefferer sir,

Okay sir, I went through #422 .
For point 1, I have renamed "pattern" to "skeleton". 
For point 2, Should I change it to "date field(see this for more information)"?
For point 3, Sure sir. But I feel after defining available skeletons in examples in this fix, it would become obvious that no non-standard pattern can be used.
For point 4, Fixed.
For point 5, Fixed.
For point 6, Sir let me know the final decision.

rxaviers commented:
@ManrajGrover, thanks for your contribution.

I'm landing the table addition. For the #422 issues, please send a new PR.
